rules/the-best-dependency-injection-container/rule.md

```diff
@@ -44,17 +42,69 @@ The top tools all contain comparable functionality. In practice which one you us
 **Important:** Unless a specific shortfall is discovered with the container your team uses, you should continue to use the same container across all of your projects, become an expert with it and invest time on building features rather than learning new container implementations.
 
 ::: bad  
-![Figure: Bad Example - Ninject was a top container but is no longer developed as actively as Autofac and Structuremap. Both Autofac and Structuremap have active communities and contributors that ensure they stay up to date with the latest changes in .Net](dic-bad.png)  
+![Figure: Bad Example - Ninject and StructureMap were top containers but are no longer actively developed. Together with Autofac, they do not support the latest version of .NET](di-container-bad.png)  
+:::
+
+```csharp
+using SSW.SugarLearning.DependencyResolver;
+using StructureMap;
+
+namespace SSW.SugarLearning.WebJob.BadgeTask
```

Comment:
Make the namespace generic?  Or remove altogether?

Reply:
Removed.